examples/prediction/xgboost-high-level-apis.ipynb

```diff
@@ -245,32 +245,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import requests\n",
-    "import json\n",
-    "json_data = {\n",
-    "    \"data\":{\n",
-    "        \"tensor\":{\n",
-    "            \"shape\":[1,37],\n",
-    "            \"values\":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37]\n",
-    "        }\n",
-    "    }\n",
-    "}\n",
-    "try:\n",
-    "    res=requests.post(endpoint.url, data={\"json\":json.dumps(json_data)})\n",
-    "    res_json = json.loads(res.text)\n",
-    "    names = res_json['data']['names']\n",
-    "    values = res_json['data']['tensor']['values']\n",
-    "    print(res_json)\n",
-    "except:\n",
-    "    pass\n",
-    "    "
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Delete endpoint"
+    "(train_X, train_y), (test_X, test_y) = read_input([\"ames_dataset/train.csv\"])\n",
+    "endpoint.predict_nparray(test_X)\n"
```

Comment:
questions for @karthikv2k : for the high level api, will Numpy arrays be the usual mode of data vars? Is this compatible with pandas?

Reply:
as an MVP numpy is fine. DS have familiarity with numpy. Definitely doing pandas support will be great and we can do it in 0.6 release.